### PR TITLE
Add verify scripts and fix some header files

### PR DESCRIPTION
**Pull request by cpanato**
Add verify scripts and fix some header files

ref: https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/266

~~Will update the presubmit jobs~~
test-infra PR: https://github.com/kubernetes/test-infra/pull/19599

/cc @justaugustus @saschagrunert @listx

### CODE REVIEW

File: bin/verify_boilerplate.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
```
Comment (saschagrunert) [inline bin/verify_boilerplate.py:1]: We still use bazel/repo-infra inside this repository, so I think we can use their boilerplate verification.

Comment (cpanato) [reply bin/verify_boilerplate.py:1]: can we just not use bazel for the verify tasks?

wdty @justaugustus ?

Comment (cpanato) [reply bin/verify_boilerplate.py:1]: adding the bazel to make all the verify checks

Comment (cpanato) [reply bin/verify_boilerplate.py]: ok, now we need to update the job again to have the bazel installed, which image should I use for that?

Comment (justaugustus) [reply bin/verify_boilerplate.py]: I'm aiming at removing bazel in this repo, so I'd prefer the checks to be in bash/python, please. You can use the ones we have in k/release.

**Comment by justaugustus** [comment]
I did some fixups and removed the bazel bits, because the presubmit is now blocking PRs.
Added the squash label as well.

/retest

**Comment by justaugustus** [comment]
Oh, right:
/lgtm
/approve

File: bin/verify_boilerplate.py (hunk 1 above)
Comment (cpanato) [reply bin/verify_boilerplate.py]: cool, thanks @justaugustus

Comment (cpanato) [reply bin/verify_boilerplate.py]: was more easy just to drop the bazel commits :)

**Pull request merged**